Sources/Sentry/SentryOptions.m

```diff
@@ -746,3 +750,29 @@ - (NSString *)debugDescription
 #endif // defined(DEBUG) || defined(TEST) || defined(TESTCI)
 
 @end
+
+@implementation SentryExperimentalOptions
+
+- (instancetype)init
+{
+
+    if (self = [super init]) {
+#if SENTRY_UIKIT_AVAILABLE
+        self.sessionReplay = [[SentryReplayOptions alloc] initWithSessionSampleRate:0
+                                                                    errorSampleRate:0];
+#endif
+    }
+    return self;
+}
+
+- (void)validateOptions:(NSDictionary<NSString *, id> *)options
```

Comment:
`m`: Do we really need that dict init option crap for experimental as well? I hate it and would like to avoid it.

Reply:
AFAIK this is the only way for hybrid to init options

Reply:
Oh now, 🙈